src/test/java/org/broadinstitute/hellbender/utils/svd/SVDUnitTest.java

```diff
@@ -1,21 +1,50 @@
 package org.broadinstitute.hellbender.utils.svd;
 
+import htsjdk.samtools.util.Log;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.ArrayUtils;
 import org.apache.commons.math3.linear.QRDecomposition;
 import org.apache.commons.math3.linear.RealMatrix;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollectionUtils;
+import org.broadinstitute.hellbender.utils.LoggingUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
 
 import java.io.File;
 import java.io.IOException;
 import java.util.List;
 
-/**
- * A class to hold some static methods useful in unit testing the SVD functionality.
- */
-final public class SVDTestUtils {
+//Note: the SVD tests were not moved to public because they use exome packages and those are in protected for now
+public final class SVDUnitTest extends BaseTest {
+    private static final File TEST_FILE_DIR = new File("src/test/resources/org/broadinstitute/hellbender/tools/exome");
+    private static final File CONTROL_PCOV_FULL_FILE = new File(TEST_FILE_DIR, "create-pon-control-full.pcov");
+    private static final File CONTROL_PCOV_GT_SV = new File(TEST_FILE_DIR, "create-pon-control-full.pcov.gt.singular_values");
+    private static final File CONTROL_PCOV_GT_V = new File(TEST_FILE_DIR, "create-pon-control-full.pcov.gt.v");
+    private static final File CONTROL_PCOV_GT_PINV = new File(TEST_FILE_DIR, "create-pon-control-full.pcov.gt.10.pinv_truncated");
 
-    private SVDTestUtils() {
+    @DataProvider(name="makeSVDers")
+    private Object[][] makeSVDers(){
+        return new Object[][]{
+                {new ApacheSingularValueDecomposer()},
+                {new OjAlgoSingularValueDecomposer()},
+                {new SparkSingularValueDecomposer(SparkContextFactory.getTestSparkContext())}
+        };
+    }
+
+    @Test(dataProvider = "makeSVDers")
+    public void testBasicFunction(final SingularValueDecomposer decomposer){
+        LoggingUtils.setLoggingLevel(Log.LogLevel.INFO);
+        try {
+            final ReadCountCollection rcc = ReadCountCollectionUtils.parse(CONTROL_PCOV_FULL_FILE);
+            final SVD svd = decomposer.createSVD(rcc.counts());
```

Comment:
Hmmm if that's the case I'm not sure the test here is needed at all.  I mean, it's not really testing an interaction with `ReadCountCollection` since `rcc` only enters via its `counts()`.  So it seems to me like this is in effect just testing the consistency of the four test files and the fact that parsing them into a `ReadCountCollection` works.